### ISSUE TITLE
Check for permission without a subject

### ISSUE THREAD

**Issue by shadoWalker89** [mblarsen/browser-acl#1]
Hi,

Laravel does not require a subject for permission checks, you can just do 

```php
// Define the permission
$gate->define('generate-statistics', function($user){
return $user->isAdmin();
});

// Check the permission
$user->can('generate-statistics');
```

I see that your package does require a subject wich will not allow doing what i described above.

**Comment by shadoWalker89** [comment]
Ok thanks

**Comment by mblarsen** [comment]
The above is not implemented. Pushing a solution now.

**Comment by mblarsen** [comment]
@shadoWalker89 please see this section https://github.com/mblarsen/browser-acl/tree/feat-global-rules#additional-parameters-and-global-rules --- does this fit your use case?

**Comment by shadoWalker89** [comment]
@mblarsen yes
Thank you

**Comment by mblarsen** [comment]
It has been published as version 0.5.0 on npm
